src/makeRobot.js

```diff
@@ -38,7 +38,69 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      const infoRobotName = `name: ${this.name}, `;
+      const infoVersion = `chip version: ${this.version}, `;
+      const infoWheels = `wheels: ${this.wheels}`;
+
+      return infoRobotName + infoVersion + infoWheels;
+    },
+    get location() {
+      const RobotName = `${this.name}: `;
+      const locationX = `x=${this.coords.x}, `;
+      const locationY = `y=${this.coords.y}`;
+
+      return RobotName + locationX + locationY;
+    },
+
+    goForward(num = 1) {
+      if (num > 0 && typeof num === 'number') {
```

Comment:
you don't need type guards
```suggestion
      if (num > 0) {
```